sample-code/python/test/helpers.py

```diff
@@ -71,6 +72,11 @@ def __save_log_type(driver, device_logger, calling_request, type):
             logcat_file.write(data_string)
 
 def report_to_sauce(session_id):
-    print("Link to your job: https://saucelabs.com/jobs/%s" % session_id)
-    passed = str(sys.exc_info() == (None, None, None))
-    sauce.jobs.update_job(session_id, passed=passed)
+    if sauce is not None:
+        print("Link to your job: https://saucelabs.com/jobs/%s" % session_id)
+        passed = str(sys.exc_info() == (None, None, None))
+        sauce.jobs.update_job(session_id, passed=passed)
+    else:
```

Comment:
do we need an else block that does nothing?

Reply:
No, but putting the comment in the else block made sense at the time.